generate-component-docs/generate.ts

```diff
@@ -9,6 +9,14 @@ const propBlacklist = ['key'];
 const tsconfigPath = path.join(__dirname, '../tsconfig.json');
 const componentsFile = path.join(__dirname, '../lib/components/index.ts');
 
+const reactNodeType =
+  'string | number | boolean | {} | ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)> | null) | (new (props: any) => Component<any, any, any>)> | ReactNodeArray | ReactPortal';
```

Comment:
This is a bit of a hack and won't work if the type changes in any way. I think just keep an eye on it as it's much cleaner with the shorthand.